### PR TITLE
Add actionable items to the issue_template

### LINKED ISSUE #29: Create an issue template
Create an issue template for -
- [x] talk proposal
- [x] Once meetup is finalized create an issue template that contains the following items

    -  Pre-meetup communication - Email and Twitter post
    -  Meetup - Description of the meetup and updated agenda
    -  Video Recording - Edit the video recording of the meetup
    -  Upload to YT - Upload the video recording to Operate First YouTube channel
    -  Send out an update - Email with an update YT link and agenda and a twitter update with the YT link

### PR DESCRIPTION
**Pull request by aakankshaduggal**
Resolve #29

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/topic_submission.md [hunk 1]
```diff
@@ -18,4 +18,12 @@ ASAP? next month? Whenever?
 Any information you'd like to provide about yourself, the presenter. This may including name, github username, organization, or project affiliations, etc. 
 
 **Additional Info**
-If applicable, please provide links to any relevant blogs, videos, git repos, etc relating to the suggested presentation. 
\ No newline at end of file
+If applicable, please provide links to any relevant blogs, videos, git repos, etc relating to the suggested presentation. 
+
+**Actionable Items**
```
Comment (oindrillac) [inline .github/ISSUE_TEMPLATE/topic_submission.md:23]: For an external contributor trying to propose their topic these Action Items may not be relevant. For example they might contribute their approved for finalized topic to the agenda document however the rest of the items seem like the organizers' responsibilities.

Should we state that here? Maybe like
```suggestion
**Actionable Items** (Following actions would be carried out by the Organizers of the Operate First Data Science Community Meetup)
```

File: .github/ISSUE_TEMPLATE/topic_submission.md [hunk 2]
```diff
@@ -18,4 +18,12 @@ ASAP? next month? Whenever?
...
+
+- [ ]  Add the topic and description to the [agenda](../../docs/content.md).
+- [ ] Send an invite via email using the [email template](../../docs/email_templates/invite_template.md) and Operate First twitter by adding a PR to [operate-first-twitter](https://github.com/operate-first/operate-first-twitter).
```
Comment (oindrillac) [inline .github/ISSUE_TEMPLATE/topic_submission.md:26]:
```suggestion
- [ ] Send an invite via email using the [email template](../../docs/email_templates/invite_template.md) and a tweet through the Operate First twitter Account by adding a PR to [operate-first-twitter](https://github.com/operate-first/operate-first-twitter).
```

File: .github/ISSUE_TEMPLATE/topic_submission.md [hunk 3]
```diff
@@ -18,4 +18,12 @@ ASAP? next month? Whenever?
...
+- [ ] Present the talk in Operate First Data Science Meetup. Follow the [format](../../docs/format.md) and record the meetup.
+- [ ] Edit the video recording of the Meetup and upload the video recording to [Operate First YouTube channel](https://www.youtube.com/channel/UCe87bwqlGoBQs2RvMQZ5_sg).
+- [ ] Send out an update via email using the [email template](../../docs/email_templates/postmeetup_update.md) and Operate First twitter by adding a PR to [operate-first-twitter](https://github.com/operate-first/operate-first-twitter).
```
Comment (oindrillac) [inline .github/ISSUE_TEMPLATE/topic_submission.md:29]:
```suggestion
- [ ] Send out meetup recording via email using the [email template](../../docs/email_templates/postmeetup_update.md) and tweet through the Operate First twitter account by adding a PR to [operate-first-twitter](https://github.com/operate-first/operate-first-twitter).   
```

**Review by oindrillac — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/topic_submission.md (hunk 1 above)
Comment (aakankshaduggal) [reply .github/ISSUE_TEMPLATE/topic_submission.md]: Sounds good :+1:

**Review by oindrillac — APPROVED** [review]
thanks for the changes!
/lgtm

**Comment by MichaelClifford** [comment]
@aakankshaduggal I'm not sure these items should be included in the topic submission template. Its a bit confusing as it seems to have two users, the topic submitter and the org admin. Why not split it up into two templates? 1) Topic submission with a checkbox for "accepted" 2) an Event template that includes the items you've outlined here?

**Comment by MichaelClifford** [comment]
/approve
:+1:

**Pull request merged**